platform/app/services/create_connection_and_syncs_service.rb

```
@@ -0,0 +1,25 @@
+    end
+
+    private
+
+    def create_connection
+      @connection_id = Connections::CreateService.new(@connector_id).call
+    end
+
+    def create_syncs
+      Syncs::CreateService.new(@connection_id).call
+    end
+  end
```

rubocop

[convention] Final newline missing.

(Layout/TrailingEmptyLines)